gamemode/modules/fadmin/fadmin/playeractions/kickban/sv_init.lua

```diff
@@ -128,6 +128,7 @@ local function Ban(ply, cmd, args)
             args[1] = targets[1]
             args[2] = args[6]
             args[3] = args[7]
+            targets = FAdmin.FindPlayer(args[1]) or targets
```

Comment:
`args[1]` is equal to `"STEAM_0"`, as shown by the check on line 126. This means that `FAdmin.FindPlayer will either find everyone or no one. Probably the former. I don't reckon this is what we want :grimacing:

Reply:
Ah of course, in that case I think `targets[1]` is clearer. I'll edit it.